modules/tinymce/src/core/main/ts/api/UndoManager.ts

```diff
@@ -58,8 +59,10 @@ const UndoManager = (editor: Editor): UndoManager => {
      */
     dispatchChange: (): void => {
       editor.setDirty(true);
+      const level = Levels.createFromEditor(editor) as UndoLevel;
+      level.bookmark = GetBookmark.getUndoBookmark(editor.selection);
```

Comment:
Minor functional change here... the the `bookmark` needs to be populated for a change event so this adds that.